hadoop-ozone/ozone-manager/src/main/java/org/apache/hadoop/ozone/protocolPB/OzoneManagerRequestHandler.java

```diff
@@ -240,11 +240,17 @@ public OMResponse handleReadRequest(OMRequest request) {
   @Override
   public OMClientResponse handleWriteRequest(OMRequest omRequest,
       long transactionLogIndex) {
-    OMClientRequest omClientRequest =
-        OzoneManagerRatisUtils.createClientRequest(omRequest);
-    OMClientResponse omClientResponse =
-        omClientRequest.validateAndUpdateCache(getOzoneManager(),
-            transactionLogIndex, ozoneManagerDoubleBuffer::add);
+    OMClientRequest omClientRequest = null;
+    OMClientResponse omClientResponse = null;
+    try {
+      omClientRequest =
+          OzoneManagerRatisUtils.createClientRequest(omRequest, impl);
+      omClientResponse = omClientRequest
+          .validateAndUpdateCache(getOzoneManager(), transactionLogIndex,
+              ozoneManagerDoubleBuffer::add);
+    } catch (IOException e) {
+      LOG.debug("Exception: " + e);
```

Comment:
Since we are not throwing `OzoneManagerRatisUtils.createClientRequest` exception, we can revert this change, isn't it?

Reply:
Done